packages/design-system/src/components/contextMenu/stories/index.js

```diff
@@ -561,14 +568,15 @@ const RightClickContextMenuContainer = styled.div`
   left: ${({ position }) => position?.x ?? 0}px;
 `;
 
-const RightClickMenuOnShapeAndBackground = ({ children }) => {
+// eslint-disable-next-line react/prop-types
+const RightClickMenuOnShapeAndBackground = ({ children, args }) => {
```

Comment:
I think the rest of the controls here need to be spread into `args`. May not need the lint rule disable if that changes
```suggestion
const RightClickMenuOnShapeAndBackground = ({ children, ...args }) => {
```

Since args are all the story's arguments right?
![image](https://user-images.githubusercontent.com/22185279/151240203-fa7550ab-da06-47c8-8f1d-cd14e68215c6.png)

Reply:
Update: this doesn't need the `args` argument. This context menu is used with a right click event to open so we can ignore passing in `args`